### DISCUSSION TITLE
Ip Spoofing with Fortio or setting a specific  IP to generate load from

### DISCUSSION THREAD

**Discussion by mbrondogutierrez** [fortio/fortio#466] [Q&A] [answered]
Hi all, is there a way to do IP spoofing with fortio? I'm trying to generate a lot of load to an external service that is request capping us at IP level, we are trying to avoid this. Since fortio will live in an AWS instance I can attach several external IP addresses to the instance but I'm having trouble trying to find if a specific IP can be attached to Fortio to produce traffic from or if we can iterate through them.
Thanks

**Comment by ldemailly** [comment, marked as answer]
So for the servers you can set which IP you bind to but that hasn't come up yet before for client. A roundabout way can probably be done using docker and setting the network for instance (setting the container to only have 1 of the IPs) it's also probably possible to use iptables, cgroups, or ipfw to achieve the same

Because I haven't had a chance to work on the distributed client side (because usually 1 client is enough to saturate servers) this hasn't come up but make sense for load distribution 

In your case though why not just allow your sourceip to not be quota limited? (exclusion list from quotas)
